tests/dashbio_demos/app_speck.py

```diff
@@ -43,67 +130,114 @@ def layout():
             ]
         ),
 
+        html.Br(),
+
+        dcc.Checklist(
+            id='speck-enable-presets',
+            options=[{'label': 'Use presets', 'value': 'True'}],
+            values=[]
+        ),
+
+        html.Br(),
+
+        dcc.Dropdown(
+            id='speck-molecule-dropdown',
+            options=[
+                {'label': 'DNA',
+                 'value': '{}dna.xyz'.format(DATAPATH)},
+                {'label': 'Caffeine',
+                 'value': '{}caffeine.xyz'.format(DATAPATH)},
+                {'label': 'Methane',
+                 'value': '{}methane.xyz'.format(DATAPATH)}
+            ],
+            value='{}dna.xyz'.format(DATAPATH)
+        ),
+
+        html.Br(),
 
         html.Div(
-            id='speck-controls',
+            id='speck-controls-preset',
+            className='speck-controls',
             children=[
+
+                "1: Select rendering style",
                 dcc.Dropdown(
-                    id='speck-molecule-dropdown',
+                    id='speck-preset-rendering-dropdown',
                     options=[
-                        {'label': 'DNA',
-                         'value': '{}dna.xyz'.format(DATAPATH)},
-                        {'label': 'Caffeine',
-                         'value': '{}caffeine.xyz'.format(DATAPATH)},
-                        {'label': 'Methane',
-                         'value': '{}methane.xyz'.format(DATAPATH)}
+                        {'label': 'Default/reset',
+                         'value': 'default'},
+                        {'label': 'Toon',
+                         'value': 'toon'},
                     ],
-                    value='{}dna.xyz'.format(DATAPATH)
+                    value='default'
                 ),
-
+                "2: Select atom style",
                 dcc.Dropdown(
-                    id='speck-preset-dropdown',
-                    options=[
-                        {'label': 'Default', 'value': 'default'},
-                        {'label': 'Ball-and-stick', 'value': 'stickball'},
-                        {'label': 'Toon', 'value': 'toon'},
-                        {'label': 'Licorice', 'value': 'licorice'}
-                    ]
-                ),
-
-                html.Hr(),
-
-                dcc.Checklist(
-                    id='speck-scroll-zoom-enable',
+                    id='speck-preset-atom-style-dropdown',
                     options=[
-                        {'label': 'Scroll to zoom', 'value': 'scrollzoom'}
+                        {'label': 'Ball-and-stick',
+                         'value': 'stickball'},
+                        {'label': 'Licorice',
+                         'value': 'licorice'}
                     ],
```

Comment:
@shammamah : don't you want to add `{'label': 'Default', 'value': 'default'}`?

Reply:
@Bachibouzouk `Default` controls the atom style and the rendering style, so I figured I'd just put it in the first dropdown and it can auto-clear the second when it's chosen. (`default` is a predefined set of parameters for the `view` prop, so I can't really merge two of them (e.g., `default` and `licorice`).

Reply:
The only issue here for me is that if you choose licorice `atom style`, then change the `rendering style` to `Toon`, once you choose `Default/reset` it doesn't keep licorice anymore. It does the action by a default convention which is implicit and not necessarily intuitive (at least not to me) ;)